src/components/AttendeeTable/index.tsx

```diff
@@ -0,0 +1,98 @@
+'use client';
+
+import { UserNamesAndIds } from '@/lib/prisma/users';
+import Paper from '@mui/material/Paper';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableFooter from '@mui/material/TableFooter';
+import TableHead from '@mui/material/TableHead';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import Typography from '@mui/material/Typography';
+import { useState } from 'react';
+
+interface Props {
+  attendees: UserNamesAndIds | null;
+  noAttendeesText: string;
+}
+
+export default function AttendeeTable({ attendees, noAttendeesText }: Props) {
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+
+  if (!attendees) return null;
+
+  if (attendees.length === 0) {
+    return (
+      <Typography variant="body2" sx={{ my: 2 }}>
+        {noAttendeesText}
+      </Typography>
+    );
+  }
+
+  const handleChangePage = (event: unknown, newPage: number) => {
```

Comment:
Can also be typed like this. This is only applicable to the `onPageChange` prop

```suggestion
  const handleChangePage = (_event: MouseEvent<HTMLButtonElement, MouseEvent> | null, newPage: number) => {
```